### PR TITLE
🐛 Ensure chunk file download

### LINKED ISSUE #205: Ensure to chunk file download
@mrnicegyu11 encountered issues (running out of memory) when downloading files via the `osparc` client. We should wrap the download method generated by the autogeneration tool so that it chunks the download and writes to file before downloading another chunk.

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
Ensure files are downloaded from the server in chunks in order to avoid the memory issues explained in #205 

## Related issue/s

- fixes #205 

## How to test
- Run `python -m pytest -k test_upload_file` from `clients/python/test/e2e`

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]

File: clients/python/src/osparc/_api_files_api.py [hunk 1]
```diff
@@ -65,24 +64,50 @@ def __getattr__(self, name: str) -> Any:
...
-            while dest_file.is_file():
-                new_name = (
-                    downloaded_file.stem
-                    + "".join(random.choices(string.ascii_letters, k=8))
-                    + downloaded_file.suffix
-                )
-                dest_file = destination_folder / new_name
-            shutil.move(downloaded_file, dest_file)
-            downloaded_file = dest_file
+        downloaded_file = Path(
```
Comment (pcrespov) [inline clients/python/src/osparc/_api_files_api.py:95]: I would download it in a temp folder and then make a copy to the destination folder when fully downloaded. 
Check for some ideas in https://www.commonwl.org/user_guide/topics/outputs.html#returning-output-files

File: clients/python/src/osparc/_api_files_api.py [hunk 2]
```diff
@@ -65,24 +64,50 @@ def __getattr__(self, name: str) -> Any:
...
+            configuration=self.api_client.configuration, timeout=timeout_seconds
+        ) as session:
+            url = urljoin(
+                self.api_client.configuration.host, f"/v0/files/{file_id}/content"
+            )
+            async for response in await session.stream(
+                "GET", url=url, auth=self._auth, follow_redirects=True
+            ):
+                response.raise_for_status()
+                with open(downloaded_file, mode="wb") as f:
```
Comment (pcrespov) [inline clients/python/src/osparc/_api_files_api.py:108]: MINOR: can also use aiofiles here?

**Review by pcrespov — APPROVED** [review]
just some thougths

File: clients/python/src/osparc/_api_files_api.py (hunk 2 above)
Comment (bisgaard-itis) [reply clients/python/src/osparc/_api_files_api.py:108]: Good point. We should do that also in the upload fcn. I can see I am jumping over fences to make things `asyncio` friendly. I will adapt

Comment (bisgaard-itis) [reply clients/python/src/osparc/_api_files_api.py]: Done

File: clients/python/src/osparc/_api_files_api.py (hunk 1 above)
Comment (bisgaard-itis) [reply clients/python/src/osparc/_api_files_api.py]: Done 😄

**Pull request merged**